src/SonarQube.Client.Tests/Api/SonarQubeService_GetQualityProfileAsync.cs

```diff
@@ -323,7 +323,7 @@ public class SonarQubeService_GetQualityProfileAsync : SonarQubeService_TestBase
   }
 }");
 
-            SetupRequest("api/qualityprofiles/changelog?qualityProfile=Sonar+way&language=cs&p=1&ps=1", @"{
+            SetupRequest("api/qualityprofiles/changelog?qualityProfile=Sonar+way&language=cs&organization=my_organization&p=1&ps=1", @"{
```

Comment:
There is an existing test for pre-6.5 that checks that the organization parameter is not passed (although it wouldn't really matter in practice if it was since it would be ignored).